addons/sale/wizard/payment_link_wizard.py

```diff
@@ -3,13 +3,26 @@
 
 from werkzeug import urls
 
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class PaymentLinkWizard(models.TransientModel):
     _inherit = 'payment.link.wizard'
     _description = 'Generate Sales Payment Link'
 
+    amount_paid = fields.Monetary(
+        currency_field='currency_id',
+        string="Already Paid",
+    )
+    show_confirmation_message = fields.Boolean(compute="_compute_show_confirmation_message")
+    is_quotation = fields.Boolean()
```

Comment:
Is this really necessary ?
We have the res_model, res_id, there is no need for a new field here.